client/ayon_maya/plugins/publish/increment_current_file_deadline.py

```diff
@@ -13,6 +13,7 @@ class IncrementCurrentFileDeadline(plugin.MayaContextPlugin):
     order = pyblish.api.IntegratorOrder + 9.0
     families = ["workfile"]
     optional = True
+    targets = ["local", "remote"]
```

Comment:
This seems to do the **opposite** of what we want to achieve... now the farm job will ALSO trigger this - but we don't want it to? We ONLY want to trigger this as we submit it to the farm (the UI publish session from the artist).

I believe this should only target `local`, right?